### PR TITLE
Privacy policy

### LINKED ISSUE #5: Create privacy policy
As per GDPR, if any personal information is stored there must be a privacy policy present to indicate what the information is used for, how it's stored and so on.

### PR DESCRIPTION
**Pull request by Jordan141**
Fixes #5

Possibly requires a small notification at the bottom to agree to these policies on page load. Needs discussion.

### CODE REVIEW

File: app/views/pages/gdprpolicy.ejs [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<%- include("../partials/header") %>
+<h1>Privacy Policy for Articleblog</h1>
+
+<p>At Articleblog, accessible from http://ec2-34-229-166-184.compute-1.amazonaws.com:8000/, one of our main priorities is the privacy of our visitors. This Privacy Policy document contains types of information that is collected and recorded by Articleblog and how we use it.</p>
```
Comment (michaelKurowski) [inline app/views/pages/gdprpolicy.ejs:4]: put here current hostname, or hostname from config.

File: app/views/pages/gdprpolicy.ejs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    <li>The right of rectification.</li> 
+    <li>The right to object.</li>
+    <li>The right of restriction.</li>
+    <li>The right to data portability</li>
+    <li>The right to withdraw consent</li>
+</ul>
+
+<h2>Log Files</h2>
+
+<p>Articleblog follows a standard procedure of using log files. These files log visitors when they visit websites. All hosting companies do this and a part of hosting services' analytics. The information collected by log files include internet protocol (IP) addresses, browser type, Internet Service Provider (ISP), date and time stamp, referring/exit pages, and possibly the number of clicks. These are not linked to any information that is personally identifiable. The purpose of the information is for analyzing trends, administering the site, tracking users' movement on the website, and gathering demographic information.</p>
```
Comment (michaelKurowski) [inline app/views/pages/gdprpolicy.ejs:34]: What do we have to do with hosting companies?

File: app/views/pages/gdprpolicy.ejs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+<p>Like any other website, Articleblog uses 'cookies'. These cookies are used to store information including visitors' preferences, and the pages on the website that the visitor accessed or visited. The information is used to optimize the users' experience by customizing our web page content based on visitors' browser type and/or other information.</p>
+
+<p>For more general information on cookies, please read <a href="https://www.cookieconsent.com/what-are-cookies/">"What Are Cookies"</a>.</p>
+
+
+
+<h2>Privacy Policies</h2>
+
+<P>You may consult this list to find the Privacy Policy for each of the advertising partners of Articleblog.</p>
```
Comment (michaelKurowski) [inline app/views/pages/gdprpolicy.ejs:46]: Do we have advertising partners?

File: app/views/pages/gdprpolicy.ejs [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+<p>Third-party ad servers or ad networks uses technologies like cookies, JavaScript, or Web Beacons that are used in their respective advertisements and links that appear on Articleblog, which are sent directly to users' browser. They automatically receive your IP address when this occurs. These technologies are used to measure the effectiveness of their advertising campaigns and/or to personalize the advertising content that you see on websites that you visit.</p>
```
Comment (michaelKurowski) [inline app/views/pages/gdprpolicy.ejs:48]: DO we use third-party ad servers?

File: app/views/pages/privacypolicy.ejs [hunk 5]
```diff
@@ -0,0 +1,60 @@
+<%- include("../partials/header") %>
+<h1>Privacy Policy of Articleblog</h1>
+
+<p>Articleblog operates the http://ec2-34-229-166-184.compute-1.amazonaws.com:8000/ website, which provides the SERVICE.</p>
```
Comment (michaelKurowski) [inline app/views/pages/privacypolicy.ejs:4]: Again, pull it from config, or hostname.

File: app/views/pages/privacypolicy.ejs [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+<p>This page is used to inform website visitors regarding our policies with the collection, use, and disclosure of Personal Information if anyone decided to use our Service, the Articleblog website.</p>
+
+<p>If you choose to use our Service, then you agree to the collection and use of information in relation with this policy. The Personal Information that we collect are used for providing and improving the Service. We will not use or share your information with anyone except as described in this Privacy Policy. Our Privacy Policy was created with the help of the <a href="https://www.privacypolicytemplate.net">Privacy Policy Template</a> and the <a href="https://www.generateprivacypolicy.com">Privacy Policy Generator</a>.</p>
+
+<p>The terms used in this Privacy Policy have the same meanings as in our Terms and Conditions, which is accessible at http://ec2-34-229-166-184.compute-1.amazonaws.com:8000/, unless otherwise defined in this Privacy Policy.</p>
```
Comment (michaelKurowski) [inline app/views/pages/privacypolicy.ejs:10]: Address

**Review by michaelKurowski — CHANGES_REQUESTED** [review]
Where did u get these texts?

File: app/views/pages/gdprpolicy.ejs (hunk 2 above)
Comment (Jordan141) [reply app/views/pages/gdprpolicy.ejs:34]: We host content and store their browser fingerprint, so we must add this part.

File: app/views/pages/gdprpolicy.ejs (hunk 3 above)
Comment (Jordan141) [reply app/views/pages/gdprpolicy.ejs:46]: It's legal coverage

File: app/views/pages/gdprpolicy.ejs (hunk 4 above)
Comment (Jordan141) [reply app/views/pages/gdprpolicy.ejs:48]: Any library imported via CDN is classified as a third party server, the wording here relating to ad servers merely serves as an example to the reader to understand the techniques and why such techniques are required.

File: app/views/pages/gdprpolicy.ejs (hunk 1 above)
Comment (Jordan141) [reply app/views/pages/gdprpolicy.ejs]: Done

**Review by michaelKurowski — APPROVED** [review]

**Pull request merged**